.golangci.yml

```diff
@@ -117,6 +117,10 @@ linters-settings:
   stylecheck:
     checks: ["all"]
 
+  revive:
+    rules:
+      - name: unexported-return
+        disabled: true
```

Comment:
Why was this disabled? 🤔

Reply:
I was hitting it on the `WithTestOpts` function which returns something unexported. We could just export that `func` if we want to keep this though

Reply:
oh makes sense! Yeah let's keep it and export the func 👍🏽